registry/README.md

```diff
@@ -1,24 +1,25 @@
 # Registry Configuration
 
-Tart supports uploading images to an OCI registry. This section describes how to get a minimal registry up and running.
+Tart supports uploading images to an OCI registry. This section describes how to setup Gitea as a container registry.
 
 ## Prerequisite
 
-- Install [Docker](https://www.docker.com/)
-- Install `htpasswd` (preinstalled on MacOS)
+```
+$ brew install colima docker
+$ colima start
```

Comment:
Should we have a `launchctl` service for the registry as well?

Reply:
Yes! (probably in another PR)